### ISSUE TITLE
Merge the latest loss functions from upstream

### ISSUE THREAD

**Issue by sadamov** [MeteoSwiss/neural-lam#5]
Joel mentioned that he added new loss functions and that the negative log likelihood performs better than mse/mae.

1. Merge the latest commit from uptream into a feature_branch
2. This will give you a new metrics.py file that contains many additional loss functions
3. Start a training with loss = negative log likelihood for 40 epochs (validation every 20 epochs)
4. Compare training loss with previous training loss curves on wandb
